cmd/ddev/cmd/global_dotddev_assets/commands/web/phive

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
I'd rather not do this custom/shell command. In general, only things that are heavily used get helper commands like this. The reason is that they clog up `ddev help` and people don't understand what they're for.

Reply:
Done